com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -49,6 +52,9 @@ public class AndroidNotificationCenter
         private static AndroidJavaObject s_CurrentActivity;
         private static bool s_Initialized;
 
+        /// <summary>
+        /// Initialize the Android notification center class.
```

Comment:
Should we call the class by name here? Something like:

> Initialize the `AndroidNotificationCenter` class.

The current phrasing makes it look like the class is called "Android notification center", which isn't accurate.

Reply:
Done